lib/shopping_cart.rb

```diff
@@ -0,0 +1,57 @@
+class ShoppingCart
+  attr_reader :name,
+              :capacity,
+              :products
+  
+  def initialize(name, capacity)
+    @name = name
+    @capacity = capacity
+    @products = []
+  end
+
+  def add_product(product)
+    @products << product
+  end
+
+  def details
+    {name: "#{name}", capacity: capacity_to_number}
+  end
+
+  def capacity_to_number
+    @capacity.delete('items').to_i
```

Comment:
We wanted this to happen when we call the method `capacity`, so I would rename this method and remove the capacity attr_reader